tests/js/sentry-test/utils.tsx

```diff
@@ -27,3 +27,23 @@ export function textWithMarkupMatcher(textMatch: string | RegExp) {
     return findTextWithMarkup(element, textMatch);
   };
 }
+
+/**
+ * Trigger onPress events on components that use react-aria (e.g. the dropdown
+ * menu). These components require more complex events than what enzyme can
+ * simulate with `.simulate('click')`. Preferably, we should use the
+ * 'user-event' library from react testing library. This function is a
+ * workaround for test files that still use enzyme. Read more:
+ * https://react-spectrum.adobe.com/react-spectrum/testing.html#triggering-events
+ */
+export function triggerPress(element) {
```

Comment:
I noticed that this function is currently only being used in the `sentry-test/dropdownMenu`, and since we agreed not to create any more enzyme files, I was wondering if we should move this helper function to the  `sentry-test/dropdownMenu` file

Reply:
hmm I put this in `utils` because it will likely be used in more (future) `react-aria` components, not just `DropdownMenuV2`. Another option is to just move `selectDropdownMenuItem` and any future utility functions for `react-aria` components into `utils`. wdyt?

Reply:
I see... in this case I think it's fine to leave this function here.

It would be super nice if we could have these tests in React testing Library :)